src/app/cli/src/coda_restarts_and_txns_holy_grail.ml

```diff
@@ -30,15 +30,22 @@ let main n () =
   (* catchup *)
   let%bind () = after (Time.Span.of_min 1.) in
   let%bind () =
-    Coda_worker_testnet.Restarts.trigger_catchup testnet ~logger ~node:2
+    Coda_worker_testnet.Restarts.trigger_catchup testnet ~logger
+      ~node:(Random.int 4 + 1)
```

Comment:
Why are we querying nodes from 1 to 4? Can we wrap this as a function? The name of the function should match with the reason why we querying nodes from 1 to 4.